scivision_test_plugin/model.py

```diff
@@ -1,6 +1,13 @@
+import os
+
+os.environ['TF_KERAS'] = '1'
+
 import numpy as np
 from skimage.filters import gaussian
 
+if os.environ.get('TF_KERAS'):
+    from classification_models.tfkeras import Classifiers
```

Comment:
Is this environmental variable necessary?

Reply:
You're right the environmental variable might be redundant. We can remove it.